src/download.html

```diff
@@ -23,8 +23,7 @@
 				<div>
 					<div>
 						<b>Platform:</b>
-						<select size="1" id="platform">
-							<option></option>
+						<select size="1" id="platform" selected="linux-amd64">
```

Comment:
If we don't know the platform, I don't think it's a good idea to select one by default.

Reply:
I'm pretty sure `linux-amd64` is by far the most used platform for Caddy (probably 65%+?) so I don't think there's any harm in defaulting to that if we can't determine the platform otherwise. The user can pick a different one if they prefer. Smooths out the UX to at least have a sensible default.

Reply:
I can see it now: people complaining that they can't run the binary they downloaded because it's the wrong platform. What if we leave it empty and show an error message if a platform is not selected?

(FWIW, I don't care too strongly about this. Just want to be thorough.)

Reply:
I feel like people who set up webservers can be trusted to understand an exec format error. Then again, one should never overestimate the intelligence of users :P still, personally I think it's fine as-is. A blank option would need a lot more handling/checking which is why I suggested removing it in the first place.